### ISSUE TITLE
TM：  The options for setting baud rate parameters are insufficient

### ISSUE THREAD

**Issue by ACTRI** [eclipse-cdt/cdt#307]
**Describe the bug**
Eclipse Target Management: Terminal
The options for setting baud rate parameters in the serial port Settings of the Terminal view are insufficient

**To Reproduce**
1. open 'Terminal Settings'  view
2. Select ‘Serial’ from ‘Connection Type’
3.click on 'Baud Rate'
4. See the options for setting baud rate parameters 
![tm1](https://user-images.githubusercontent.com/121931500/223051162-0f175c8e-97d6-472b-a9ed-a20e6911a72a.jpg)

**Expected behavior**
Add the common baud rate parameter, set the default value to 115200, and set the baud rate to editable
![tm2](https://user-images.githubusercontent.com/121931500/223051188-7f42e9c6-63c9-44bb-b7f4-1b0739887b03.jpg)

**Desktop (please complete the following information):**
 - OS: windows7
 - TM-3.6.1

**Comment by jonahgraham** [comment]
Thanks for the enhancement request @ACTRI. Fortunately I already added this functionality a while ago in da0d1d7df6e5db543ef808c2cc99e037fecb7805. TM-3.6.1 is now quite old and any version of the terminal released after June 2020 has these fixes. 

Here is a screenshot on Linux:

![image](https://user-images.githubusercontent.com/679236/223198313-1318a372-0ad7-4609-b04c-3f0dfc9e85b2.png)

PS. 

> open 'Terminal Settings' view

I am not sure where exactly the GUI you have screenshoted has come from. I don't have a "Terminal Settings" view, and my screenshot doesn't quite match what you have shown. The dialog I screenshoted came from the "Open a Terminal" command in the "Terminal" view:

![image](https://user-images.githubusercontent.com/679236/223199056-c81cf008-2921-43ae-a835-b571b6912092.png)
